CRM/Core/Config.php

```diff
@@ -373,7 +373,11 @@ private function _initialize($loadFromDB = TRUE) {
       echo 'You need to define CIVICRM_TEMPLATE_COMPILEDIR in civicrm.settings.php';
       exit();
     }
-
+    $upload_dir    = wp_upload_dir();
```

Comment:
I missed this,  The current structure places the WP settings file in a different structure than Drupal and Joomla.  Rather than putting proper structure here, where should we define it?